cache/models/samples.js

```diff
@@ -666,7 +666,7 @@ module.exports = {
       return redisOps.getHashPromise(aspectType, aspectName);
     })
     .then((aspObj) => {
-      if (!aspObj) {
+      if (!aspObj || (aspObj.isPublished == 'false')) {
```

Comment:
probably do something like  `if (!aspObj || !aspObj.isPublishe)`

Reply:
aspObj.isPublished is a string. !'true' == !'false' => false